taxi/tests/test_models.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from taxi.models import Manufacturer, Car
+
+
+class ManufacturerModelTest(TestCase):
+    def test_manufacturer_str(self) -> None:
+        manufacturer = Manufacturer.objects.create(
+            name="test name",
+            country="test country"
+        )
+        self.assertEqual(
+            str(manufacturer),
+            f"{manufacturer.name} {manufacturer.country}"
+        )

```

Comment:
In the `test_manufacturer_str` method, ensure that the expected string format `f"{manufacturer.name} {manufacturer.country}"` matches the actual `__str__` method implementation in the `Manufacturer` model. If the `__str__` method uses a different format, adjust the test accordingly.